artifacts/etcdstorage/crd.yaml

```diff
@@ -19,3 +19,25 @@ spec:
             name:
               type: string
               maxLength: 59 # because of service name, explained above.
+        spec:
+          properties:
+            caCertConfigMap:
+              type: array
+              items:
+                type: object
+                required: ["name", "namespace"]
+                properties:
+                  name:
```

Comment:
can they be empty? What is their format? (hint: pattern with regexps)

Reply:
> can they be empty? What is their format? (hint: pattern with regexps)

@sttts going to stick on it or will you take a followup issue?

Reply:
Created follow-up issue #44

Reply:
Fixed in this PR.